src/create-client.ts

```diff
@@ -107,7 +107,7 @@ export class Client {
     const searchParams = new URLSearchParams(window.location.search);
     if (isRedirectCallback(this.#redirectUri, searchParams)) {
       await this.#handleCallback();
-    } else {
+    } else if (document.cookie.includes("workos-has-session=")) {
```

Comment:
in the (far) future, this check might be better off inside of `#refreshSession` (that would prevent `getAccessToken()` calls from making api calls that cannot succeed).

we can't do that right now though because that make it seem like current sessions have ended (users don't have the cookie yet).